app/src/main/java/org/oppia/app/drawer/NavigationDrawerItem.kt

```diff
@@ -0,0 +1,19 @@
+package org.oppia.app.drawer
+
+import org.oppia.app.R
+
+/** Enum to store the menu of [NavigationDrawerFragment] and get menu by id. */
+enum class NavigationDrawerItem(val value: Int) {
+  HOME(R.id.nav_home), HELP(R.id.nav_help);
+
+  companion object {
+    fun valueFromInt(v: Int): NavigationDrawerItem {
```

Comment:
I suggest ‘valueFromNavId’ for the name since ‘int’ is really ambiguous (and I would actually expect to correspond to the enum’s ordinal).

Reply:
Refactored as valueFromNavId